spec/factories/quest_keys.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :quest_key do
+    quest ""
+    key ""
+    event "MyString"
```

Comment:
This is not a valid event. Create different factories for different events (aka `traits`) and test accordingly.